recipes/ofeli/5.x/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rmdir, replace_in_file, rename
+
+required_conan_version = ">=1.53.0"
+
+
+class OfeliConan(ConanFile):
+    name = "ofeli"
+    description = "An Object Finite Element Library"
+    license = "LGPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://ofeli.org/index.html"
+    topics = ("finite-element", "finite-element-library", "finite-element-analysis", "finite-element-solver")
+
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            raise ConanInvalidConfiguration("Ofeli only supports Linux")
+        if self.settings.compiler != "gcc":
+            raise ConanInvalidConfiguration("Ofeli only supports GCC")
```

Comment:
This is not explicit in the upstream, or I didn't find it documented. Do you have any real reference?

Reply:
Added a comment for it now:
> include/linear_algebra/LocalVect_impl.h:251:42: error: cannot initialize return object of type 'OFELI::Element *' with an lvalue of type 'const OFELI::Element *'

Probably not too difficult to fix, but I would leave it to someone else to take care of. Did not see any PRs for it either.